tests/test_discounts.py

```diff
@@ -246,3 +249,24 @@ def test_sale_applies_to_correct_products(product_class):
         net=5, currency='USD')
     with pytest.raises(NotApplicable):
         sale.modifier_for_product(sec_variant.product)
+
+
+@pytest.mark.django_db
+def test_get_category_variants_and_prices_product_with_many_categories(cart, default_category, product_in_stock):
+    """Test error: get_category_variants_and_prices return duplicated items. It cause double(or more) percentage discount"""
```

Comment:
This will become meaningless as soon as we merge this branch. Please drop the docstring.